feat(graphcache): allow for defining inline-fragment/fragment-definition client controlled nullability directives (#3502)
urql-graphql/urql: .changeset/happy-peas-sin.md

---
'@urql/exchange-graphcache': minor
---

> Allow for defining `@_optional` and `@_required` on fragment-definitions and inline-fragments

Reviewer: ```suggestion
Allow `@_optional` and `@_required` to be placed on fragment definitions and inline fragments
```
